content/html/concepts/favicon/favicon.md

```diff
@@ -0,0 +1,29 @@
+---
+Title: 'Favicon'
+Description: 'A favicon is a small icon that is displayed in the browser's address bar and tab title for a website.'
+Subjects:
+  - 'Web Development'
+  - 'Web Design'
+Tags:
+  - 'Favicon'
```

Comment:
The tags listed need to be on the list in the [tags.md](https://github.com/Codecademy/docs/blob/main/documentation/tags.md) file. Either we need to remove "Favicon" from here, or add it to the tags.md file in this PR.